### PR TITLE
Implement back button on login screen

### PR DESCRIPTION
**Pull request by ronniebeggs**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
- Created a back button component that reroutes the user to the starting page
- Styled the button using react native icons
- Replaced logo placeholder

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy!"
back-button

## How to review
[//]: # "Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"
- Run the app and log out if you're already signed in
- Navigate from the starting screen to the login screen
- Press the back button in the top left corner; it should take you back to the starting screen

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."
- Ensure that styling is consistent with the rest of the app
- Abstract the button as a component for use on other screens

### Online sources
[//]: # "Optional - copy links to any tutorials or documentation that was useful to you when working on this PR"
Using react-native-vector-icon library: https://aboutreact.com/react-native-vector-icons/

### Related PRs
[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"
[Annie's navigation structure PR](https://github.com/calblueprint/vouchers4veggies/pull/19)
[Noah's auth navigation PR](https://github.com/calblueprint/vouchers4veggies/pull/22)

[//]: # "This tags the project leader as a default. Feel free to change, or add on anyone who you should be in on the conversation."
🥦 CC: @sauhardjain

### CODE REVIEW

File: assets/Components.tsx [hunk 1]
```diff
@@ -67,3 +67,14 @@ export const LargeInputField = styled.TextInput`
   padding: 10px;
   width: 317px;
 `;
+
+export const RootNavBackButton = styled.TouchableOpacity`
+  text-align: center;
+  align-items: center;
+  width: 20%;
```
Comment (sauhardjain) [inline assets/Components.tsx]: Good practice to use `px` instead of percentages while styling buttons.

File: assets/Components.tsx [hunk 2]
```diff
@@ -67,3 +67,14 @@ export const LargeInputField = styled.TextInput`
...
+  font-weight: 500;
```
Comment (sauhardjain) [inline assets/Components.tsx]: Remove the `font-weight` because it doesn't affect the icons.

File: src/screens/auth/LoginScreen.tsx [hunk 3]
```diff
@@ -54,12 +57,13 @@ export default function LoginScreen() {
...
-      <LogoContainer>
-        <View style={{ backgroundColor: 'black', width: 50, height: 59.29 }}>
-          <Text style={{ color: 'white' }}>{`\n  Logo`}</Text>
-        </View>
-      </LogoContainer>
+      <BackButtonContainer>
+        <RootNavBackButton onPress={() => navigation.navigate('Start')}>
+          <Text>
+            <Icon name="left" size={14} color="#525454" /> Back
+          </Text>
```
Comment (sauhardjain) [inline src/screens/auth/LoginScreen.tsx]: Replace the `<Text>` tag with the pre-styled `DarkGrayText`.

File: src/screens/auth/LoginScreen.tsx [hunk 4]
```diff
@@ -54,12 +57,13 @@ export default function LoginScreen() {
...
-      {/* logo placeholder */}
-      <LogoContainer>
-        <View style={{ backgroundColor: 'black', width: 50, height: 59.29 }}>
-          <Text style={{ color: 'white' }}>{`\n  Logo`}</Text>
-        </View>
-      </LogoContainer>
+      <BackButtonContainer>
+        <RootNavBackButton onPress={() => navigation.navigate('Start')}>
+          <Text>
+            <Icon name="left" size={14} color="#525454" /> Back
```
Comment (sauhardjain) [inline src/screens/auth/LoginScreen.tsx]: Use the darkGray color from our global `Colors.ts` file. You can do this by setting the color as `${Colors.darkGray}`.

File: assets/Components.tsx [hunk 5]
```diff
@@ -67,3 +67,14 @@ export const LargeInputField = styled.TextInput`
...
+  font-size: 14px;
+  line-height: 19px;
+  color: #525454;
```
Comment (sauhardjain) [inline assets/Components.tsx]: We shouldn't have any text styling as part of a styled button. Shift this into a styled Text container if required.

**Review by sauhardjain — CHANGES_REQUESTED** [review]
GREAT PR @ronniebeggs! Left a few comments — mostly nits about styling practices. Should be ready to merge once these are resolved.

**Review by sauhardjain — APPROVED** [review]
LGTM

File: src/screens/auth/LoginScreen.tsx [hunk 6]
```diff
@@ -13,16 +14,19 @@ import {
...
   VerticalSpacingButtonContainer,
   WhiteText,
+  BackButtonContainer,
+  DarkGrayText,
 } from './styles';
 
-export default function LoginScreen() {
+export default function LoginScreen({
+  navigation,
+}: AuthStackScreenProps<'Start'>) {
```
Comment (wangannie) [inline src/screens/auth/LoginScreen.tsx:29]: Pretty sure this should be 'Login' instead 

@ronniebeggs @sauhardjain
